components/odh-notebook-controller/Dockerfile.konflux

```diff
@@ -43,12 +32,16 @@ RUN microdnf install -y shadow-utils &&\
 ## Create a non-root user with UID 1001
 RUN useradd --uid 1001 --create-home --user-group --system rhods
 
+## Set workdir directory to user home
 WORKDIR /home/rhods
 
-COPY --from=builder /workspace/notebook-controller/manager .
-COPY --from=builder /workspace/notebook-controller/third_party/license.txt third_party/license.txt
```

Comment:
was never particularly clear on the importance of this license.txt, probably ok to leave it out